themes/default/layouts/challenge/list.html

```diff
@@ -0,0 +1,76 @@
+{{ define "hero" }}{{ partial "hero.html" (dict "title" "Challenges") }}{{ end }}
+{{ define "main" }}
+    <section class="max-w-4xl container mx-auto py-8 px-4 text-center text-lg">
+        <p>
+            We think Pulumi is an amazing IaC tool that increases developer velocity and handles the scale of the cloud with ease. Try it out for yourself by taking a Pulumi
+            Challenge, and tell us if you think we're wrong! You'll build something cool, and we'll send you some super secret swag in return. We promise it's better than a
+            t-shirt.
+        </p>
+    </section>
+
+    <section id="challenges" class="w-full relative overflow-visible">
+        <!-- Only do the 2x2 layout at tablet width (md) and up. -->
+        <ul class="md:flex flex-wrap list-none">
+            <li class="md:w-1/2">
+                <div>
+                    <div class="w-full p-4 my-2">
+                        <div class="card text-center py-16 px-8 relative bg-white">
+                            <h4>Startup in a Box</h4>
```

Comment:
In the future having a custom graphic for each of these boxes would make it seem a bit more exciting.